ncm-postgresql/src/main/pan/components/postgresql/schema.pan

```diff
@@ -2,37 +2,29 @@
 # ${developer-info}
 # ${author-info}
 
-############################################################
-#
-# type definition components/postgresql
-#
-#
-#
-#
-############################################################
-
-
 declaration template components/postgresql/schema;
 
-include {'quattor/schema'};
+include 'quattor/types/component';
 
-function pgsql_is_hba_db = {
+function postgresql_is_hba_db = {
     # Check cardinality and type of argument.
     if (ARGC != 1 || !is_string(ARGV[0]))
         error("usage: is_asndate(string)");
```

Comment:
Error message has incorrect function name. (Use FUNCTION global to guarantee correctness?)